go/internal/flag/flag.go

```diff
@@ -0,0 +1,46 @@
+// Package flag is an internal package to allow us to gracefully transition
```

Comment:
In theory we need a copyright section here (though we are in a bit of flux around what that will look like).